### DISCUSSION TITLE
Tooltip header shows time instead of date

### DISCUSSION THREAD

**Discussion by LoganMcnulty** [TanStack/react-charts#218] [Q&A] [answered]
I have a simple multi-dataset with dates as the x-axis.

Tooltip shows the time from the date object instead of the date (the axis correctly shows the date).

I have looked into using custom tooltips, but would rather not go through implementing an entire custom tooltip just to change the header. The current tooltip layout looks perfect.

![image](https://user-images.githubusercontent.com/52426539/125526762-2a5c1931-4356-42a7-9caf-db84312a0285.png)

Thoughts? And thank you.

Logan

**Comment by tannerlinsley** [comment, marked as answer]
You need to either use the UTC axis or alter your dates to use midnight.

**Reply by LoganMcnulty** [reply]
Giving these a shot now, TY

**Reply by LoganMcnulty** [reply]
Confirmed that both methods work. Midnight set to "time", otherwise set to "UTC" and the time seems to round.

IMO current date format is too undescriptive. A param for mm/dd/yyyy would be huge.

![image](https://user-images.githubusercontent.com/52426539/125537024-9388ab23-3bb5-4642-b1d6-5491e19e1ce0.png)

Thanks again

**Reply by eversadia** [reply]
> You need to either use the UTC axis or alter your dates to use midnight.

Is there a way to change the header into something like "2021-10-20 05:40"
